### PR TITLE
Added new condition

### PR DESCRIPTION
**Pull request by taongbayan01**
ssadasdasfe

### CODE REVIEW

**Review by calacrito — COMMENTED** [review]
Nice add another condition

**Review by calacrito — COMMENTED** [review]
add something

File: pull.project.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-if(val = "Yes"){
+if(val == "Yes" && val == "y"){
   
 }else{
```
Comment (calacrito) [inline pull.project.js]: Add something in this block

File: pull.project.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-if(val = "Yes"){
+if(val == "Yes" && val == "y"){
```
Comment (calacrito) [inline pull.project.js]: Add something

**Review by calacrito — COMMENTED** [review]
additional statements

File: pull.project.js (hunk 2 above)
Comment (taongbayan01) [reply pull.project.js]: Ok got it

**Review by calacrito — CHANGES_REQUESTED** [review]
add another one in the else block

**Review by calacrito — APPROVED** [review]
Good

**Pull request merged**